mne/io/reference.py

```diff
@@ -193,6 +193,12 @@ def add_reference_channels(inst, ref_channels, copy=True):
                      'unit': FIFF.FIFF_UNIT_V,
                      'coord_frame': FIFF.FIFFV_COORD_HEAD,
                      'loc': np.zeros(12)}
+        # see if "zeroth" EEG electrode present in dig points
+        dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+        if len(dig_loc) == 1:
+            chan_info['loc'][:3] = dig_loc[0]['r'][:]
```

Comment:
`len(dic_loc)` will always be <= 1 for fif-files, since `'ident'` can only be 0 for a single electrode. I placed the if there to catch a case where no `dig_loc`s are found (have no experience with other file format sources in this respect)